plugins/addP5.js

```diff
@@ -0,0 +1,43 @@
+var fs = require('fs');
+var path = require('path');
+var through = require('through2');
+var cheerio = require('cheerio');
+
+// This constructor function will be called once per format
+// for every build. It received a plugin registry object, which
+// has .add(), .before() and .after() functions that can be used
+// to register plugin functions in the pipeline.
+
+var Plugin = function(registry){
+  registry.after('markdown:convert', 'addP5:insert', this.addFrames);
+};
+
+Plugin.prototype = {
+
+  addFrames: function(config, stream, extras, cb) {
+    if(config.format == "html") {
+
+    stream = stream.pipe(through.obj(function(file, enc, cb) {
+        if(!file.$el) file.$el = cheerio.load(file.contents.toString());
+          // Loop through all figures to replace with iframes
+          file.$el('figure[data-pde-sketch^="http"]').each(function(i, el) { 
+            var jel = file.$el(this);
+            var source = jel.attr('data-pde-sketch');
```

Comment:
Let's change this to `data-p5-sketch`! Or just `p5-sketch`? "pde" is a reference to Processing IDE which isn't relevant here so p5 will be more clear!